### PR TITLE
Filtering

### PR DESCRIPTION
**Pull request by georgecrawford**

### CODE REVIEW

File: client/components/search/_main.scss [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+	margin-bottom: 10px;
+}
+
+/**
+ * Pagination
+ */
+
+.search-pagination__header {
+	@include oTypographySansData(m);
+	margin-bottom: 15px;
```
Comment (laurieboyes) [inline client/components/search/_main.scss]: Could this (and maybe other spacing bits) use Next's `$spacing-unit` sass variable? https://github.com/Financial-Times/n-ui/blob/6c0d735930168dc6d3c24eac4bd7158ec9337f02/env.scss#L10

File: client/components/search/_main.scss [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+}
+
+.search-filtering__taxonomy-label {
+	width: 100%;
+}
+
+.search-filtering__taxonomy-input {
+	width: 100%;
+	padding: 5px;
+	font-size: 1.1em;
```
Comment (laurieboyes) [inline client/components/search/_main.scss]: Is there any chance we can fudge this to be a standard Origami font size? https://github.com/Financial-Times/o-typography#sans

File: client/components/search/_main.scss [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+}
+
+.search-filtering__submit {
+	text-align: center;
+	margin-top: 30px;
```
Comment (laurieboyes) [inline client/components/search/_main.scss]: Could this be `$spacing-unit * 2` ?

File: client/components/search/main.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+		button.classList.add('is-closed');
+		button.classList.remove('is-open');
+	}
+};
+
+module.exports = {
+	init: (flags) => {
+
+		if (!flags.get('searchFilterResults')) {
```
Comment (laurieboyes) [inline client/components/search/main.js]: Flag use looks good to me! 👍

File: client/components/search/main.js [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+			return;
+		}
+
+		const button = document.querySelector('.search-filtering__toggle');
+		const panel = document.querySelector('.search-filtering__panel');
```
Comment (laurieboyes) [inline client/components/search/main.js]: Could we decouple the javascript element selector stuff from the BEM-style layout classes? I like to have an additional class like  `.js-search-filter-toggle` or something just for JS's use

**Comment by georgecrawford** [comment]
Thanks @laurieboyes. :)

**Pull request merged**